docs-content/general/changelog/changelog-28.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Changelog 28 (3/6)
+slug: changelog-28
+---
+
+## Related Resources
+
+No error takes place in a vacuum.
+
+Highlight is built around the concept of connecting all of your sessions, errors, logs, and traces.
+
+We're tightening up that integrated approach by detecting those connections proactively. Now, when you're investigating an error, you'll see buttons for related sessions, logs, and traces.
+
+If those buttons are active, you can click through to see the related resource and dig deeper into the error's cause.
+
+And of course, we have similar buttons on our Session Replay page, as well as links back to the relevant session from the Traces page.
+
+![Related Resources buttons](/images/changelog/28/related-resources.png)
+
+## Search query evolution
```

Comment:
Not sure if we want to call out these short cuts on the query search - contains and matches (i.e. regex)
![Screenshot 2024-03-06 at 11 17 15 AM](https://github.com/highlight/highlight/assets/17744174/3418175b-5f9b-4c3f-aed4-c32ac3fcb71e)

Reply:
Thanks for the images!